css/styles.css

```
@@ -315,53 +315,46 @@ td p span {
     bottom:52px;
     left: 80px;
 }
-/*
- *Section three
- */
+
+/*Section three*/
```

After every comment line there is a space.
```suggestion
/* Section three */
```

Another example:
``` // Comment ```